src/bin/ion/commands/inspect.rs

```diff
@@ -494,9 +511,66 @@ impl<'a, 'b> IonInspector<'a, 'b> {
                     use MacroExprKind::*;
                     match invocation.kind() {
                         EExp(eexp_arg) => self.inspect_eexp(depth + 1, eexp_arg)?,
-                        EExpArgGroup(_) => todo!("e-exp arg groups"),
+                        EExpArgGroup(arg_group) => {
+                            self.inspect_eexp_arg_group(depth + 1, arg_group)?
```

Comment:
🪧 There's now basic support for arg groups. E-expressions in the arg group do not yet expand.